bentoml/cli/bento.py

```diff
@@ -237,3 +245,112 @@ def retrieve(bento, target_dir):
         safe_retrieve(bento_service_bundle_path, target_dir)
 
         click.echo('Service %s artifact directory => %s' % (name, target_dir))
+
+    @cli.command(
+        help='Containerize given Bento into a Docker image',
+        short_help="Containerize given Bento into a Docker image",
+    )
+    @click.argument(
+        "bento", type=click.STRING, callback=parse_bento_tag_callback,
+    )
+    @click.option('--push', is_flag=True)
+    @click.option(
+        '--docker-repository',
+        help="Prepends specified Docker repository to image name.",
+    )
+    @click.option(
+        '-u', '--username', type=click.STRING, required=False,
+    )
+    @click.option(
+        '-p', '--password', type=click.STRING, required=False,
+    )
+    def containerize(bento, push, docker_repository, username, password):
+        """Containerize specified BentoService.
+
+        BENTO is the target BentoService to be containerized, referenced by its name
+        and version in format of name:version. For example: "iris_classifier:v1.2.0"
+
+        `bentoml containerize` command also supports the use of the `latest` tag
+        and will automatically use the last built version of your Bento.
+
+        You can also optionally provide a `--push` flag, which will push the built
+        image to the Docker repository specified by the `--docker-repository`.
+
+        If you would like to push to Docker Hub, `--docker-repository` can just be
+        your Docker Hub username. Otherwise, the tag should include the hostname as
+        well. For example, for Google Container Registry, the `--docker-repository`
+        would be `[HOSTNAME]/[PROJECT-ID]`
```

Comment:
Add to the doc: `for private docker registry, this would be 'myregistrydomain.com:5000/'`

Reply:
This parameter is a little bit confusing because the `--docker-repository` parameter is not really the `repository` concept in docker registry.  I'm thinking just calling this `--docker-image-tag-prefix` instead?  

Alternatively we can make it two separate parameters: `--docker-image-tag` and `--docker-registry-hostname`. For docker hub user, the --docker-image-tag here can be `{dockerhub_username}/IrisClassifier`, we will use this tag name instead of the BentoService name. For private docker registry, this can be `--docker--registry-hostname=mydockerregistry.com:5000/` and for google cloud user: `--docker--registry-hostname=HOSTNAME  --docker-image-tag=PROJECT-ID/IrisClassifier`.

Reply:
A third option is to just provide one `--docker-image-tag` option, and for users using `push`, they are responsible for prefixing the tag with a hostname. It seems easier to understand and matches the experience of using docker build from CLI. thoughts?

Reply:
I think the third option here makes sense! I'll work on making that change